modules/administration-guide/partials/con_che-server-components.adoc

```diff
@@ -0,0 +1,23 @@
+[id="{prod-id-short}-server-components_{context}"]
+= {prod-short} server components
```

Comment:
Neither of the build previews render what is written here in this con_che-server-componets.adoc:

https://61571a657e455f0cfdf33983--eclipse-che-docs-pr.netlify.app/che-7/administration-guide/server-components/

https://61571a657e455f0cfdf33983--eclipse-che-docs-pr.netlify.app/che-7/administration-guide/server-components/